test/evaluation/inputDescriptorFilterEvaluationHandler.spec.ts

```diff
@@ -74,13 +93,20 @@ describe('inputDescriptorFilterEvaluationHandler tests', () => {
     message2.payload = {"result": [], "valid": true};
     const evaluationClient: EvaluationClient = new EvaluationClient();
     const evaluationHandler: EvaluationHandler = new InputDescriptorFilterEvaluationHandler(evaluationClient);
-    evaluationHandler.handle(presentationDefinition, new VP(presentation));
+    evaluationHandler.handle(presentationDefinition, presentation);
     expect(evaluationClient.results).toEqual([message0, message1, message2]);
   });
 
   it(`input descriptor's constraints.fields.path does not match`, () => {
     const inputCandidates: unknown = getFile('./test/dif_pe_examples/vp/vp_general.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.